app/controllers/api/internal/tax_documents_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Api::Internal::TaxDocumentsController < Api::Internal::BaseController
+  before_action :authenticate_user!
+  before_action :ensure_tax_center_enabled
+  after_action :verify_authorized
+
+  def index
+    authorize :balance
```

Comment:
Used this existing policy since the authorization logic is basically the same, but I am open to add a standalone policy for this stuff.